### ISSUE TITLE
Trailing comments break enablesid.conf

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ype/pulledpork#96]
```
What steps will reproduce the problem?
1. Create enablesid.conf containing a single line like '1:xxxx" with a sid that 
you'd like to enable.
2. Run pulled pork, note during the enablesid phase that 1 sid is processed.
3. Add a trailing comment to your file, like so:

1:xxxx # I'm a comment describing why this sid has been enabled

What is the expected output? What do you see instead?
When running PP after step three, thousands of sids will be enabled, presumably 
because of a bad regex that doesn't expect a trailing comment.

What version of the product are you using? On what operating system?
0.6.1 on RHEL6

Please provide any additional information below.
Trailing comments are VERY useful.  Please fix poste-haste.
```

Original issue reported on code.google.com by `m...@nyu.edu` on 17 Oct 2011 at 10:03

**Comment by GoogleCodeExporter** [comment]
```
A second engineer in my office just dropped 3 hours troubleshooting this bug 
before asking me what was going on.  I suspect that other people are being bit 
by it regularly and simply not figuring out the root cause of the problem.
```

Original comment by `mikeloc...@gmail.com` on 12 Jan 2012 at 7:49

**Comment by GoogleCodeExporter** [comment]
```
This is slated for s fix in the upcoming release.. I don't have a solid date 
for that, but note that it's on the list.
```

Original comment by `Cummin...@gmail.com` on 13 Jan 2012 at 3:10
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Going to set to allow trailing comments only.. i.e. no comments in-between 
entries on a single line.
```

Original comment by `Cummin...@gmail.com` on 26 Jan 2012 at 6:40

**Comment by GoogleCodeExporter** [comment]
```
Fix Committed Rev #237
```

Original comment by `Cummin...@gmail.com` on 26 Jan 2012 at 7:03
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 124 has been merged into this issue.
```

Original comment by `Cummin...@gmail.com` on 16 Jan 2013 at 1:03
